drivers/display/display_stm32_ltdc.c

```diff
@@ -252,14 +252,23 @@ static int stm32_ltdc_init(const struct device *dev)
 #if defined(CONFIG_SOC_SERIES_STM32F4X) || defined(CONFIG_SOC_SERIES_STM32F7X)
```

Comment:
Should be `#if defined(CONFIG_SOC_SERIES_STM32F4X)` as you're fully duplicating the code.